vllm/config/__init__.py

```diff
@@ -357,9 +357,11 @@ class ModelConfig:
     logprobs_mode: LogprobsMode = "raw_logprobs"
     """Indicates the content returned in the logprobs and prompt_logprobs.
     Supported mode:
-    1) raw_logprobs, 2) processed_logprobs, 3) raw_logits, 4) processed_logits.
-    Raw means the values before applying logit processors, like bad words.
-    Processed means the values after applying such processors.
+    1) raw_logprobs, 2) processed_logprobs, 3) final_logprobs, 4) raw_logits,
```

Comment:
We can just have the processed_logprobs  / processed_logits as final_logprobs / final_logits. No need to introduce the new mode.